recipes_source/distributed_comm_debug_mode.rst

```diff
@@ -0,0 +1,197 @@
+Using CommDebugMode
+=====================================================
+
+**Author**: `Anshul Sinha <https://github.com/sinhaanshul>`__
+
+Prerequisites:
+
+- `Distributed Communication Package - torch.distributed <https://pytorch.org/docs/stable/distributed.html>`__
```

Comment:
I think it's included with PyTorch, so we probably don't need to state this explicitly?